framework/meta/src/cargo_toml_contents.rs

```diff
@@ -55,6 +83,88 @@ impl CargoTomlContents {
             .to_string()
     }
 
+    pub fn package_edition(&self) -> String {
+        self.toml_value
+            .get("package")
+            .expect("missing package in Cargo.toml")
+            .get("edition")
+            .expect("missing package name in Cargo.toml")
+            .as_str()
+            .expect("package name not a string value")
+            .to_string()
+    }
+
+    pub fn mvx_dependency_version(&self) -> String {
+        let toml_value = self.dependency("multiversx-sc").unwrap().to_owned();
+
+        if let Value::Table(table) = toml_value {
+            if let Some(version) = table.get("version") {
+                return version.to_string().replace('\"', "");
```

Comment:
Extract this into a function that removes the quotes. It is used in several places.